tests/vm_test.go

```diff
@@ -284,11 +285,14 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		It("[test_id:3162]should ignore kubernetes and kubevirt annotations to VMI", func() {
```

Comment:
For a follow up, this test should be consider for removal in favor of a simple unit test or checking it as part of a wider test (e.g. the test that checks that an annotation is passed to the VMI could also check that other annotations are not passed).